### PR TITLE
Milestone3pr

### PR DESCRIPTION
**Pull request by LadyEr0n**
something

### CODE REVIEW

File: Assets/Scripts/DetectLookedAtInteractive.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        LookedAtInteractive = interactive;
+
+        if (objectWasDetected)
+        {
+            //Debug.Log($"Player is looking at: {hitInfo.collider.gameObject.name}");
+            interactive = hitInfo.collider.gameObject.GetComponent<IInteractive>();
+        }
+
+
```
Comment (JaredBronstein) [inline Assets/Scripts/DetectLookedAtInteractive.cs:69]: This space is unnecessary. One space is all you need

**Review by JaredBronstein — APPROVED** [review]
Overall pretty good, some minor formatting issues with spaces where they don't belong and no space where they should be. When formatting keep similar lines of code next to each other without a space between them and very different lines a space away. If it's doing the same thing with just different values then keep em close. Functions though should always have a space between them though, just to make them easier to identify since they're usually so vastly different from each other in terms of what they do that the space is necessary. Approved!

File: Assets/Scripts/DetectLookedAtInteractive.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
```
Comment (dantogno) [inline Assets/Scripts/DetectLookedAtInteractive.cs:6]: Remove extra lines / whitespace unless you are systematically adding extra whitespace / line breaks to enhance the readability of your document (for example, by having a blank line separating distinct elements).

File: Assets/Scripts/DetectLookedAtInteractive.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        get { return lookedAtInteractive; }
+        private set
+        {
+            bool isInteractiveChanged = value != lookedAtInteractive;
+            if (isInteractiveChanged)
+            {
+                lookedAtInteractive = value;
+                LookedAtInteractiveChanged?.Invoke(lookedAtInteractive);
+            }
+
```
Comment (dantogno) [inline Assets/Scripts/DetectLookedAtInteractive.cs:37]: Remove extra lines / whitespace unless you are systematically adding extra whitespace / line breaks to enhance the readability of your document (for example, by having a blank line separating distinct elements).

**Pull request merged**